app/admin/targets.rb

```diff
@@ -0,0 +1,37 @@
+ActiveAdmin.register Target do
+  actions :index, :show
+
+  controller do
+    def scoped_collection
+      super.includes(:user, :topic)
+    end
+  end
+
+  index do
+    id_column
+    column :title
+    column 'Radius (m)', &:radius
+    column :lat
+    column :lng
+    column :matched
+    column :user
+    column :topic
+    column :created_at
+
+    actions
+  end
+
+  show do
+    attributes_table do
+      row :title
+      row 'Radius (m)', &:radius
```

Comment:
maybe we can use i18n to override this column head

Reply:
Good idea!